spec/features/publish/courses/new_tda_course_spec.rb

```diff
@@ -137,6 +137,29 @@
     and_i_do_not_see_the_degree_awarding_option
   end
 
+  scenario 'back links when choosing a teacher degree apprenticeship' do
+    given_i_am_authenticated_as_a_school_direct_provider_user
+    and_the_tda_feature_flag_is_active
+    when_i_visit_the_courses_page
+    and_i_click_on_add_course
+    and_i_choose_a_primary_course
+    and_i_choose_a_primary_age_range
+    then_i_see_the_degree_awarding_option
+    when_i_choose_a_degree_awarding_qualification
```

Comment:
Can we leave a space between the whens and thens to make it easier to read?

Reply:
Done